controls/cis_ubuntu2404.yml

```diff
@@ -2126,13 +2126,14 @@ controls:
     notes: TODO. Rule does not seem to be implemented, nor does it map to any rules in ubuntu2204 profile.
 
   - id: 5.4.2.4
-    title: Ensure root account access is controlled (Automated)
+    title: Ensure root account access is controlled
```

Comment:
I would prefer to keep the `(Automated)` in the title to be consistent with other rules

Reply:
Fixed